.gitignore

```diff
@@ -61,4 +61,4 @@ bld/
 *.VC.opendb
 
 # Visual C++ Solution
-*.sln
+*.VC.db

```

Comment:
Definitely, the*.VC.db have to be ignored and good that we are not ignoring now the solution file.